src/scss/_button.scss

```diff
@@ -0,0 +1,23 @@
+$color: #ACACAC;
+$grey: #D3D3D3;
```

Comment:
I can kinda get behind `$grey` if everyone is going to agree on the same shade of grey, but `$color` is _way_ too generic a name. Since that variable is defined at the top of the file, outside of any selector block, anyone else who uses a Sass variable named `$color` will end up clobbering your color. :(